config.json

```diff
@@ -16,6 +16,7 @@
       "/api/v1/addresses*",
       "/api/v1/treasury",
       "/api/v1/receipts*",
+      "/api/plugins/faucet/*",
```

Comment:
this makes no sense in the mainnet config

Reply:
free coins for everyone :tada: